frontend/pages/service/add.js

```diff
@@ -1,40 +1,38 @@
 import AddServiceForm from '../../components/service/addServiceForm';
 import { useRouter } from 'next/router';
 import { useState } from 'react';
-import { http } from "../../utils/http";
-
-const apiPath = '/api/v1';
```

Comment:
Question: did this cause any issue for the deployment? 
I used this so we can easily change the api version later. Am I missing something about the api path use?

Reply:
That's a good observation, initially I just reverted everything from that breaking PR and re-added things line by line to check where it breaks, tried different things like resetting my db, etc. to know whats the root cause. From that process I probably didn't revert that portion.

For your api versioning concerns, they are valid but you can probably just append it once in my http method instead of duplicating it in many different files. It makes it easier to change too.